### ISSUE TITLE
RuntimeException in FFProbe.php line 244: Unable to probe with any external link/URL

### ISSUE THREAD

**Issue by SunChero** [PHP-FFMpeg/PHP-FFMpeg#249]
this is specifically an issue with windows ...
i got it working fine on linux .. but in windows .. even if the binaries are working fine and give the expected output .. the class will throw an exception.
on the same windows machine ... copying the file to local disk ...and giving the local path to ffprobe work just fine !
both machines are connected with no proxy 

something to note BTW i realised ffprobe doesnt behave the same when used with single quote as input 
in other words :  ffprobe   http:/somthing  will work both on linux and windows .. but   ffprobe  'http://something'  wont work on windows !!

**Comment by matrunchyk** [comment]
@SunChero would be nice to take a look at your full backtrace.
